### PR TITLE
Add `.deleteAll` method to `Map` and `WeakMap`

### PR DESCRIPTION
**Pull request by zloirock**
https://github.com/tc39/proposal-collection-methods/issues/4#issuecomment-362102639 + #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,15 +66,20 @@ New methods are added to `Map.prototype`.
   * `Map.prototype.filter(predicate, thisArg)`
   * `Map.prototype.mapValues(fn, thisArg)`
   * `Map.prototype.mapKeys(fn, thisArg)`
-
 * New methods:
+  * `Map.prototype.deleteAll(...elements)` - similar to `Set.prototype.deleteAll`.
```
Comment (ljharb) [inline README.md]: should there be a `setAll`, then?

Comment (zloirock) [reply README.md]: Something like that. `Set.prototype.set` accept 2 arguments, so it will not work by the same way as `.addAll` or `.deleteAll`.

Comment (ljharb) [reply README.md]: i'd expect `setAll` on Maps to accept entries as arguments.

**Comment by zloirock** [comment]
What keeps this PR from merging?

**Pull request merged**